src/settings/components/ProxyServerSettingsConfig/ProxyServerSettingsView/testResources.js

```diff
@@ -0,0 +1,61 @@
+const input = {
+  'name': 'stringTemplates[0]',
+  'value': {
+    'id': 'b03c2a7e-5ed3-4d58-8cdc-7e8c3fdf9d74',
+    'dateCreated': '2023-01-10T14:01:36Z',
+    'rule': 'http://localhost:3000/settings/local-kb-admin/proxy-server-settings',
+    'context': {
+      'id': '2c91809c859beb1f01859bf328ab0047',
+      'value': 'urlproxier',
+      'label': 'urlProxier'
+    },
+    'lastUpdated': '2023-01-10T15:59:02Z',
+    'name': 'test',
+    'idScopes': [{
+      'label': 'Oxford Academic Journals',
+      'value': '64d8acbf-5964-4223-bc5c-83c3659ef843'
+    }]
+  },
+  'onBlur': 'ƒ () {}',
```

Comment:
To add to the above, functions should ALWAYS be replaced with jest.fn(), not sent as strings

Reply:
That said, this time they're within input, which we just shouldn't be sending at all, so this will be a moot point